CMakeLists.txt

```diff
@@ -0,0 +1,137 @@
+cmake_minimum_required(VERSION 3.1)
+project(libdill VERSION 1.6 LANGUAGES C)
+
+include(CheckSymbolExists)
+include(CheckFunctionExists)
+
+file(GLOB sources *.c dns/*.c)
+include_directories(${PROJECT_SOURCE_DIR} "${PROJECT_SOURCE_DIR}/dns")
+set_source_files_properties(dns/dns.c PROPERTIES COMPILE_FLAGS -std=c99)
```

Comment:
Will not work for some compilers (e.g. Visual Studio).

Reply:
I copied most of this from libmill because libdill is just the next iteration of the libmill project.